src/RedisEnterpriseCache/RedisEnterpriseCache.Autorest/examples/Invoke-AzRedisEnterpriseCacheForceDatabaseLinkToReplicationGroup.md

```diff
@@ -0,0 +1,22 @@
+### Example 1: {{ Add title here }}
```

Comment:
Please finish the examples and regenerated the module

Reply:
Added

Reply:
@revanthballa1188 You should add the example here and the files of `docs` are all generated.

Reply:
Got it, Updated now!